docs/docusaurus.config.js

```diff
@@ -43,6 +43,26 @@ const config = {
         disableInDev: false,
       },
     ],
+    [
+      require.resolve('docusaurus-lunr-search'),
+      {
+        indexBaseUrl: true,
+        includeRoutes: [
+          '/docs/api/API/*',
+          '/docs/general/*',
+          '/docs/general/**/*',
+          '/docs/next/api/API/*',
+          '/docs/next/general/*',
+          '/docs/next/general/**/*',
+          '/docs/0.11.6/api/API/*',
+          '/docs/0.11.6/general/*',
+          '/docs/0.11.6/general/**/*',
+          '/docs/0.12.2/api/API/*',
+          '/docs/0.12.2/general/*',
+          '/docs/0.12.2/general/**/*',
```

Comment:
Do we want to hardcode the versions here? Seems like this would be something that is easily forgotten to be updated with the next releases...